roles/etcd/tasks/join_etcd-events_member.yml

```diff
@@ -22,13 +23,14 @@
       {%- endfor -%}
 
 - name: Join Member | Ensure member is in etcd-events cluster
-  shell: "{{ bin_dir }}/etcdctl --no-sync --endpoints={{ etcd_events_access_addresses }} member list | grep -q {{ etcd_events_access_address }}"
+  shell: "{{ bin_dir }}/etcdctl --no-sync member list | grep -q {{ etcd_events_access_address }}"
   register: etcd_events_member_in_cluster
   changed_when: false
   check_mode: no
   tags:
     - facts
   environment:
+    ETCDCTL_ENDPOINTS: "{{ etcd_events_access_addresses }}"
```

Comment:
Same for `ETCDCTL_API` here

Reply:
Missed it, adding that in latest commit.